code/src/Dropdown.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import "./Dropdown.css"
+
+export const Dropdown = (props) => {
+
+    return (
+        <label className="label">
+            <h3> {props.label} </h3>
+            <select className="dropdown"
+                onChange={event => props.onChanged(event.target.value)}
+                value={props.selectedValue}
+            >
+                <option value="">{props.title}</option>
+                {props.values.map((item, index) => {
+                    return (
+                        <option key={index} value={item}>
```

Comment:
Smart to use index as the key. 🤓 I've read that you may use the item index as a key if you don't have any other stable options.